pkg/ccl/changefeedccl/changefeedbase/settings.go

```diff
@@ -346,5 +346,5 @@ var Quantize = settings.RegisterDurationSettingWithExplicitUnit(
 	settings.ApplicationLevel,
 	"changefeed.resolved_timestamp.granularity",
 	"the granularity at which changefeed progress is quantized to make tracking more efficient",
-	time.Duration(metamorphic.ConstantWithTestRange("changefeed.resolved_timestamp.granularity", 0, 0, 10))*time.Second,
+	time.Duration(metamorphic.ConstantWithTestRange("changefeed.resolved_timestamp.granularity", 1, 0, 10))*time.Second,
```

Comment:
maybe we want to take this opportunity to add a valid range to it, eg `settings.DurationWithMinimum(0)`

Reply:
done

Reply:
FYI it seems like for duration settings, it will already validate that it's non-negative if no explicit validation is specified https://github.com/cockroachdb/cockroach/blob/06521cdaff7970aa1072ba57729ee9dd6251aac7/pkg/settings/duration.go#L211-L214